Src/Add-AzureDtlVM.ps1

```diff
@@ -1,64 +1,142 @@
+<#
+.SYNOPSIS 
+    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
+
+.DESCRIPTION
+    It allows the creation of the VMs inside a specific lab. This script needs to be scheduled for the creation of the VMs of each lab. 
+
+.PARAMETER LabName
+    Mandatory. Name of Lab.
+
+.PARAMETER VMCount
+    Mandatory. Number of VMs to create with this execution.
+
+.PARAMETER ImageName
+    Mandatory. Name of base image in lab.
+
+.PARAMETER TotalLabSize
+    Mandatory. Desired total number of VMs in the lab.
```

Comment:
Add what this is used for, i.e. if the lab already contains N machines, it won't create more.